perun-openapi/openapi.yml

```diff
@@ -16215,6 +16235,21 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/servicesManager/blockServiceOnDestinations:
```

Comment:
Same with this and 'facility-service-destinations', we need to block/unblock destinations on different services